nntrainer/tensor/manager.h

```diff
@@ -365,7 +390,7 @@ class Manager {
    * will require full allocation than reusing memory allocated with inference
    * mode.
    */
-  void initializeTensors(bool training);
+  void initializeTensors(bool training, unsigned int max_exec_order);
```

Comment:
Could you update doxygen? (also for line 639, 644)

Reply:
Updated. 639 and 644 are removed in the next PR, so they are skipped.

Reply:
Okay